README.md

```diff
@@ -49,86 +48,12 @@ Notes:
 - `**`: Snapshot management including creating/deleting snapshot and restoring volume from snapshot.
 - `***`: Only supports Snapshot Management on Kubernetes v1.17+ because snapshot features goes into Beta on this version.
 
-### Installation
-This guide will install CSI plugin in the *kube-system* namespace of Kubernetes v1.16+. You can also deploy the plugin in other namespace. 
-
-- Set Kubernetes Parameters
-  - For Kubernetes v1.16
-    - Enable `--allow-privileged=true` on kube-apiserver, kube-controller-manager, kube-scheduler, kubelet.
-    - Enable (Default enabled) [Mount Propagation](https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation) feature gate。
-    - Enable (Default enabled) `--feature-gates=CSINodeInfo=true,CSIDriverRegistry=true,ExpandCSIVolumes=true` option on kube-apiserver, kube-controller-manager, kube-scheduler, kubelet
-    - Enable `--read-only-port=10255` on kubelet
-  - For Kubernetes v1.17+
-    - Enable `--allow-privileged=true` on kube-apiserver, kube-controller-manager, kube-scheduler, kubelet.
-    - Enable (Default enabled) [Mount Propagation](https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation) feature gate。
-    - Enable (Default enabled) `--feature-gates=CSINodeInfo=true,CSIDriverRegistry=true,ExpandCSIVolumes=true,VolumePVCDataSource=true` option on kube-apiserver, kube-controller-manager, kube-scheduler, kubelet
-    - Enable `--read-only-port=10255` on kubelet
```

Comment:
This part of those lines is very useful, can be used in the readme of the helm chart.